Testing/UseCaseTests/UseCaseTests.cpp

```diff
@@ -0,0 +1,149 @@
+// only need to run pythonEnvSetup once or it continuously appends to paths
+// FIXME: This is a problem if you only need to execute a single test (same problem in ShapeworksTests)
+TEST(UseCaseTests, setup)
+{
+  pythonEnvSetup();
+  ASSERT_TRUE(true);
+}
+
+//---------------------------------------------------------------------------
+TEST(UseCaseTests, ellipsoid)
+{
+  run_use_case("ellipsoid",
+               "Output/ellipsoid/shape_models/32/ellipsoid_00.isores.center.com.aligned.cropped.tpSmoothDT_local.particles");
+}
+
+//---------------------------------------------------------------------------
+TEST(UseCaseTests, ellipsoid_mesh)
+{
+  run_use_case("ellipsoid_mesh",
+               "Output/ellipsoid_mesh/shape_models/32/ellipsoid_00_local.particles");
+}
+
+//---------------------------------------------------------------------------
+TEST(UseCaseTests, ellipsoid_fd)
+{
+  run_use_case("ellipsoid_fd",
+               "Output/ellipsoid_fd/shape_models/128/ellipsoid_19.isores.center.com.aligned.cropped.tpSmoothDT_local.particles");
+}
+
+//---------------------------------------------------------------------------
+TEST(UseCaseTests, ellipsoid_cut)
+{
+  run_use_case("ellipsoid_cut",
+               "Output/ellipsoid_cut/shape_models/16/ellipsoid_00.tpSmoothDT_local.particles");
+}
+
+//---------------------------------------------------------------------------
+TEST(UseCaseTests, ellipsoid_evaluate)
+{
+  run_use_case("ellipsoid_evaluate",
+               "Output/ellipsoid/evaluation/compactness/scree.txt");
+}
+
+
+//---------------------------------------------------------------------------
+TEST(UseCaseTests, lumps)
+{
+  run_use_case("lumps",
+               "Output/lumps/shape_models/32/lump0_0_local.particles");
+}
+
+//---------------------------------------------------------------------------
+TEST(UseCaseTests, left_atrium)
+{
+  run_use_case("left_atrium",
+               "Output/left_atrium/shape_models/32/CARMA0046.laendo_no_veins.isores.center.pad.com.aligned.cropped.tpSmoothDT_local.particles");
+}
+
+
+//---------------------------------------------------------------------------
+TEST(UseCaseTests, femur)
+{
+  run_use_case("femur",
+               "Output/femur/shape_models/32/m03_L_femur.isores.pad.com.center.aligned.clipped.tpSmoothDT_local.particles");
+}
+
+//---------------------------------------------------------------------------
+TEST(UseCaseTests, femur_mesh)
+{
+  run_use_case("femur_mesh",
+               "Output/femur_mesh/shape_models/32/m03_L_femur_local.particles");
+}
+
+//---------------------------------------------------------------------------
+TEST(UseCaseTests, femur_cut)
+{
+  run_use_case("femur_cut",
+               "Output/femur_cut/shape_models/32/m03_L_femur.isores.pad.com.center.aligned.tpSmoothDT_local.particles");
+}
+
+//---------------------------------------------------------------------------
+TEST(UseCaseTests, deep_ssm)
+{
+  /// TODO: Disabled, doesn't run on windows GH
+  return;
```

Comment:
Is there a GitHub issue for this?

Reply:
As we discussed in the developer meeting, I think we just need to drop it for now.  @jadie1 was going to look into whether it's feasible to have a CPU based tiny test for this.